Allow script name to contain a dash or dot in the NAME section

skirmess opened the pull request:
https://github.com/cpants/Module-CPANTS-Analyse/issues/34

charsbar commented:
Thanks. I think your idea is nice, but this implementation seems a bit too loose. (.pm files (and probably .pod files as well) shouldn't be affected by this, right?) Could you make it to see a $file's extention? I would appreciate it as well if you could add a test case in xt/kwalitee/has_abstract_in_pod.t, or just let me know which distribution you found this issue in.

skirmess commented:
I've changed the commit to not allow dash and dot in `.pm` and `.pod` files but I'm unsure if the change meets your code style. Feel free to adjust it or let me know if you want me to implement it differently.

The problem is with https://cpan.metacpan.org/authors/id/S/SK/SKIRMESS/App-ReportPrereqs-0.001.tar.gz

I've added it to `xt/kwalitee/has_abstract_in_pod.t` but I couldn't figure out what the second argument is (`0` or `1`), therefore I've picked `0`.

charsbar commented:
Thanks.
